charts/incubator/privatebin/values.yaml

```
@@ -0,0 +1,60 @@
+image:
+  repository: privatebin/pdo
+  pullPolicy: IfNotPresent
+  tag: 1.4.0@sha256:5fe4b94286dcadbd0e576418f619205463030754e257863ec430eb7eb3e957fd
+
+env:
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#timezone-settings
+  TZ: "{{ .Values.TZ }}"
+  PHP_TZ: "{{ .Values.TZ }}"
+
+securityContext:
+  # This image supports running in read-only mode. Using this reduces the attack
+  # surface slightly, since an exploit in one of the images services can't
+  # overwrite arbitrary files in the container. Only /tmp, /var/tmp, /var/run &
+  # /srv/data may be written into.
+  # See also: https://github.com/PrivateBin/docker-nginx-fpm-alpine#running-the-image
+  readOnlyRootFilesystem: true
+  privileged: false
+  allowPrivilegeEscalation: false
+
+service:
+  main:
+    protocol: HTTP
+    ports:
+      main:
+        targetPort: 8080
+        port: 8080
+
+probes:
+  liveness:
+    path: "/"
+    port: 8080
+  readiness:
+    path: "/"
+    port: 8080
+  startup:  # TODO disable?
+    path: "/"
+    port: 8080
```

```suggestion
probes:
  liveness:
    path: "/"
  readiness:
    path: "/"
  startup:  # TODO disable?
    path: "/"
```
The default port is the port of the `main` service.

Why you want to disable startup probe?

---

I was just blindly copy and pasting the k8s sample from the PrivateBin docs, and they omitted the startup probe.

---

Cool, generally it doesn't need to be disabled unless it's failing for some reason